x/concentrated-liquidity/position_test.go

```diff
@@ -2245,3 +2246,298 @@ func (s *KeeperTestSuite) TestNegativeTickRange_SpreadFactor() {
 
 	s.clk.InitGenesis(s.Ctx, *export)
 }
+
+func (s *KeeperTestSuite) TestTransferPositions() {
+	// expectedUptimes are used for claimable incentives tests
+	expectedUptimes := getExpectedUptimes()
+
+	errTolerance := osmomath.ErrTolerance{
+		AdditiveTolerance: osmomath.NewDec(1),
+		// Actual should be less than expected, so we round down
```

Comment:
Would be great to add why this is the case

Reply:
Added explanation of why this is the case https://github.com/osmosis-labs/osmosis/pull/6623/commits/c96b5053c286f34080cbc3746c55a6d24eaedd09